### ISSUE TITLE
notes about using with webpack

### ISSUE THREAD

**Issue by PhantomPayne** [lacona/lacona-phrase#1]
Using this module with webpack has a couple of gotchas.

1) Because of the reference to `../package` you got to make sure you have the json-loader included.

``` javascript
module: {
    loaders: [
        { test: /\.json$/, loader: 'json' },
    }
}
```

2) If you want to use a custom phrase(That's probably why you're including this!), you'll need to resolve/alias `lacona-phrase`.(at least this was how I got it to work).  It is possible that it loads two copies of `lacona-phrase` and since lacona uses `instanceof` to verify that you're inheriting from Phrase it fails because it's looking at two different copies. 

``` javascript
resolve: {
    alias: {
        "lacona-phrase": path.resolve('./node_modules/lacona-phrase'),
    },
},
```

**Comment by brandonhorst** [comment]
Yeah, Lacona was built and tested entirely using Browserify, which handles both of those cases automatically. Could you submit a PR for the readme with this info in a readable format for Webpack users?
